flycheck-haskell.el

```diff
@@ -257,7 +257,11 @@ buffer."
                 (append .extensions .languages
                         flycheck-ghc-language-extensions))
     (setq-local flycheck-ghc-args
-                (append .other-options flycheck-ghc-args))))
+                (append .other-options
+                        (cons "-hide-all-packages"
+                              (mapcar (apply-partially #'concat "-package ")
+                                      .dependencies))
```

Comment:
Ah, it does work :confused: GHC has a very strange approach to parsing arguments…